pyincore/hazardservice.py

```diff
@@ -1054,3 +1055,13 @@ def search_hurricanewf(self, text: str, skip: int = None, limit: int = None, tim
         r = self.client.get(url, params=payload, timeout=timeout, **kwargs)
 
         return self.return_http_response(r).json()
+
```

Comment:
@navarroc here i added the method meant to be replaced by real API calls once we have that endpoint